debezium-connector-db2/src/main/java/io/debezium/connector/db2/Db2SchemaChangeEventEmitter.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.db2;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.pipeline.spi.SchemaChangeEventEmitter;
+import io.debezium.relational.Table;
+import io.debezium.schema.SchemaChangeEvent;
+import io.debezium.schema.SchemaChangeEvent.SchemaChangeEventType;
+
+/**
+ * {@link SchemaChangeEventEmitter} implementation based on SQL Server.
+ *
+ * @author Jiri Pechanec
```

Comment:
Can you add your name where feasible?

Reply:
Updated names where DB2-specific changes were introduced.